File: bubble_sort.py
def bsort(A):
    for i in range(len(A)):
        swapped = False
        for j in range(0,len(A)-i-1):
            if A[j]>A[j+1]:
                temp = A[j]
                A[j] = A[j+1]
                A[j+1] = temp
                swapped = True
        if not swapped:
            break
    print(A)
L = [i for i in range(0,10000)]
bsort(L)

File: BubbleSort.c
#include <math.h>
#include <stdio.h>

void bsort(int a[], int n){
    int i, j,temp;
    for(i=0; i<n-1; ++i){
        for(j = 0; j<n-i-1; ++j){
            if (a[j] > a[j+1]){
                temp = a[j];
                a[j] = a[j+1];
                a[j+1] = temp;

            }
        }
    }
}

void print(int a[], int n){
    int i;
    for ( i = 0; i<n; ++i){
        printf("%d", a[i]);
    printf("\n");
    }
}

int main(){
    int L[] = {78,432,37,6,0,9};
    int n = sizeof(L)/sizeof(L[0]);
    bsort(L,n);
    print(L,n);
    return 0;
}